doc/source/user_guide/data_types.rst

```diff
@@ -124,6 +124,56 @@ may write::
    >>> plt.imshow(out)
 
 
+Working with OpenCV
+===================
+
+It is possible that you may need to use an image created using ``skimage`` with
+OpenCV or vice versa. OpenCV image data can be accessed (without copying) in
+NumPy.
+OpenCV uses BGR (instead of scikit-image's RGB) for color images, and its
+dtype is uint8 by default (See `Image data types and what they mean`_).
+
+``skimage`` provides various utility functions to convert between different data
```

Comment:
This paragraph can perhaps be removed; it doesn't seem particularly relevant at this point.